OracleDatabase/OracleRealApplicationClusters/README.md

```diff
@@ -0,0 +1,474 @@
+
+Section 2   : Building the Oracle RAC Database Docker Install Images
+
+Section 3   : Creating the Docker GI and RAC container
+
+Section 4   : Adding RAC Node using Docker container
+
+Section 5   : Connecting to RAC Database
+
+Section 6   : Env Variables for RAC 1st Node container
+
+Section 7   : Env Variables for RAC 2nd Node container
+
+Section 8   : License
+
+Section 9   : Copyright
+
+
+Section 1 : Pre-requsites for RAC on Docker
+================================
+IMPORTANT: You must make the changes specified in this section (customized for your environment) before you proceed to the next section.
+
+- For better performance, it is good to use BTRFS file system for Docker storage on the Docker host.
+- Install and configure Docker Engine on Oracle Linux 7 to run RAC on Docker.
+- Each container that you will deploy as part of your cluster must satisfy the minimum hardware requirements of the RAC and GI software. An Oracle RAC database is a shared everything database. All data files, control files, redo log files, and the server parameter file (SPFILE) used by the Oracle RAC database must reside on shared storage that is accessible by all the Oracle RAC database instances. You must provide block devices shared across the hosts.  If you don't have shared block storage, you can, for testing purpose only, use the Oracle rac-storage-server image to deploy a docker container providing NFS-based sharable storage.  Note: If you are planning to use the RAC storage Container for shared storage, you must create the RAC Storage Container before starting the containers. 
+
+- Allocate block devices of at least 50GB for OCR/Voting and databases files. You can refer Oracle 12.2 GRID and database install guides for details on recommended number of devices and total storage size. 
+
+- You must configure the following addresses manually in your DNS:
+
+	- Public IP address for each container
+
+	- Private IP address for each container
+
+	- Virtual IP address for each container
+
+	- Three single client access name (SCAN) addresses for the cluster.
+
+	For details, please refer to following guide:
+	https://docs.oracle.com/database/122/LADBI/LADBI.pdf
+
+- You need to plan your private and public network for containers before you start installation. You can create a network bridge on every host so containers running within that host can communicate with each other.  For example, create rac_pub1_nw for the public network (172.15.1.0/24) and rac_priv1_nw (192.168.17.0/24) for a private network. You can use any network subnet for testing. However, in this document we reference the public network on 172.15.1.0/24 and the private network on 192.168.17.0/24.
+
+		#docker network create --driver=bridge --subnet=172.15.1.0/24 rac_pub1_nw
+
+		#docker network create --driver=bridge --subnet=192.168.17.0/24 rac_priv1_nw
+
+		Note: You must run  RAC on Docker on multi-host using MACVLAN Docker driver (see Docker documentation).
```

Comment:
If you need to use the `MACVLAN` driver, then please provide example commands on how to configure this.